Library/Formula/cucumber-cpp.rb

```diff
@@ -0,0 +1,55 @@
+class CucumberCpp < Formula
+  desc "Support for writing Cucumber step definitions in C++"
+  homepage "https://cucumber.io"
+  url "https://github.com/cucumber/cucumber-cpp/archive/v0.3.tar.gz"
+  version "0.3"
+  sha256 "1c0f9949627e7528017bf00cbe49693ba9cbc3e11087f70aa33b21df93f341d6"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+
+  def install
+    system "cmake", "-DCUKE_DISABLE_GTEST=on",
+                    "-DCUKE_DISABLE_CPPSPEC=on",
+                    "-DCUKE_DISABLE_FUNCTIONAL=on",
+                    "-DCUKE_DISABLE_BOOST_TEST=on",
+                    "."
+    system "cmake", "--build", "."
+    include.install "include/cucumber-cpp"
+    lib.install Dir["src/*.a"]
+  end
+
+  test do
+    home = ENV["HOME"]
+    ENV["GEM_HOME"] = home
+    ENV["BUNDLE_PATH"] = home
+    ENV.prepend_path "PATH", "#{home}/bin"
```

Comment:
I update my code with your advice but the error is still there: when starting `cucumber` (with `system "cucumber"`) I have an execution failure:

```
==> cucumber
Failed to execute: cucumber
Error: cucumber-cpp: failed
Failed executing: cucumber
/usr/local/Library/Homebrew/formula.rb:1477:in `block in system'
/usr/local/Library/Homebrew/formula.rb:1414:in `open'
/usr/local/Library/Homebrew/formula.rb:1414:in `system'
/usr/local/Library/Formula/cucumber-cpp.rb:52:in `block in <class:CucumberCpp>'
/usr/local/Library/Homebrew/formula.rb:1323:in `block in run_test'
/usr/local/Library/Homebrew/extend/fileutils.rb:35:in `mktemp'
/usr/local/Library/Homebrew/formula.rb:1319:in `run_test'
/usr/local/Library/Homebrew/test.rb:28:in `block in <main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/timeout.rb:66:in `timeout'
/usr/local/Library/Homebrew/test.rb:27:in `<main>'
```

I manage to make it work by using the full path anyway (system testpath/"cucumber")so it is not a big deal.